hooks_template.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+"""
+This file defines the required template for a given hooks-module to be used with the import_video.py 
+script. The `register_camera` and `post_video_content` functions are required, but the `set_hook_data`
+is optional and will only be called if it exists
+"""
+
+def set_hook_data(data_dict):
+    """
+    let the importer pass in arbitrary key-value pairs for use by the hooks program
+    (we tend to use this to accept plan data or user-account information)
+    data_dict: a python list/dictionary of values passed in from the video importer script, passed in by the
+                user under the --hook_data_json argument.
+    """
+    pass
+
+
+def register_camera(camera_name, host=None, port=None):
+    """
+    arguments:
+        camera_name   - the name of the camera (as parsed from the filename)
+        host          - the URI/IP address of the segmenter being used
+        port          - the port to access the webserver of the segmenter
+    returns: this function returns a dictionary describing the new camera, including the camera ID
```

Comment:
it does return a dictionary.. an actual python dictionary. We don't pass json around in python that's only used to injest config from the user.